### ISSUE TITLE
Failed to create extension btree_gist.control is missing on centos 7-8

### ISSUE THREAD

**Issue by pompon79** [citusdata/pg_auto_failover#558]
Fresh install on centos 7/8, i have installed postgres sql server version 12 and pg-auto-failover14_12.x86_64 trying to initialise monitor i get following error

```
09:55:19 125682 ERROR Failed to create extension "pgautofailover"[58P01]: ERROR:  could not open extension control file "/usr/pgsql-12/share/extension/btree_gist.control": No such file or directory

postgres log 
2020-12-18 09:55:18.881 CET [125664] ERROR:  could not open extension control file "/usr/pgsql-12/share/extension/btree_gist.control": No such file or directory
```

This is correct there is no such a file in the given directory. 
I have tested this on ubuntu and i get no error however required file btree_gist.control can be found in /usr/share/postgres/12/extension. 
After some research and googling i installed package postgresql12-contrib.x86_64 and this fixes the issue. Maybe you can update documentation and add to "Dependencies" section contrib package on centos/rhel family.

**Comment by DimCitus** [comment]
Hi @pompon79 ; thanks for your feedback. Are you using community RPM packages, or the packages that our team provide over at https://packagecloud.io/citusdata/community ?

**Comment by pompon79** [comment]
Hello

```
Installed Packages
Name         : pg-auto-failover14_12
Version      : 1.4.1
Release      : 1.el8
Architecture : x86_64
Size         : 2.3 M
Source       : pg-auto-failover14_12-1.4.1-1.el8.src.rpm
Repository   : @System
From repo    : citusdata_community
```

Postgres packages are installed as advised via official postgres site ex

https://www.postgresql.org/download/linux/redhat/

```
# Install the repository RPM:
sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm

# Install PostgreSQL:
sudo yum install -y postgresql12-server
```

**Comment by devrimgunduz** [comment]
Hi,

So, there are two issues in here:

* The *community*  package is lacking -contrib dependency. I fixed it in 1.4.1 update. Thanks for the report.
* You are using Citus' pg_auto_failover package.  That is fine, but the problem you reported is in their package.

Given that you are already using PostgreSQL community repo, you can switch to our packages. yum/dnf install pg_auto_failover_12 will do the trick for you.
 
Regards, Devrim

**Comment by hanefi** [comment]
We fixed our packages as well. For future reference, please consider doing either of the following:
1. Updating to a more recent version with the fix
2. Installing postgresqlXY-contrib packages manually
